lingetic-spring-backend/Dockerfile

```
@@ -2,23 +1,24 @@
 WORKDIR /app
 
-# Install findutils to provide xargs which is required by gradle but not provided by the base image
-RUN yum update -y && yum install -y findutils
-
 COPY gradlew .
 COPY gradle gradle
 COPY build.gradle.kts .
 COPY settings.gradle.kts .
 
-COPY config/application.properties config/application.properties
+RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
```

Hadolint (2.12.0)

[warning] Quote this to prevent word splitting.

(SC2046)